### PR TITLE
Feature/education inbox letters UI

### PR DESCRIPTION
**Pull request by FernandoBoza**
## Description
Built the UI for /education/education-inbox

## Original issue(s)
[Original PR](https://github.com/department-of-veterans-affairs/vets-website/pull/20164)

## Testing Notes
From the list of acceptable test users >
- After setting up AIO, create a user on LTS apache with service record of more than 10 year, honorable, and end of terms.
- After go to postman and test users ssn on the POST endpoint **Claimant Info Exist in LTS and assiociate**
- After add user to DBeaver and SQLDeveloper, in DBeaver you have to currently manually add the submission_date, after claimant id and status 
- After testing your user on the GET letters endpoint, you should get a PDF.
- After you can checkout my branch, run yarn install, then yarn watch.

## Testing done
- [x] In progress
- [ ] Done

## Definition of done
- [X] Events are logged appropriately
- [X] Documentation has been updated, if applicable
- [X] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by timwright12 — APPROVED** [review]

File: src/applications/education-inbox/components/Layout.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Breadcrumbs from '@department-of-veterans-affairs/component-library/Breadcrumbs';
+
+const Layout = ({ children, clsName = '' }) => {
+  const classNa = `main ${clsName}`;
+  return (
+    <>
+      <Breadcrumbs>
```
Comment (nathanielbuck) [inline src/applications/education-inbox/components/Layout.jsx:8]: There is `<va-breadcrumbs>` to use now.

**Review by nathanielbuck — APPROVED** [review]

File: src/applications/education-inbox/components/Layout.jsx (hunk 1 above)
Comment (FernandoBoza) [reply src/applications/education-inbox/components/Layout.jsx:8]: ill update it now

**Review by it-harrison — APPROVED** [review]

File: src/applications/education-inbox/containers/InboxPage.jsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+          includes details of our decision, including benefit program, amount,
+          and level.
+        </p>
+        <div className="vads-u-display--flex vads-u-align-items--center">
+          <a
+            className="vads-u-flex--1"
+            download
+            href={`${environment.API_URL}/meb_api/v0/claim_status`}
+          >
+            <i className="fa fa-download vads-u-display--inline-block vads-u-margin-right--1" />
```
[the remark this answers is not in the document]
Comment (FernandoBoza) [reply src/applications/education-inbox/containers/InboxPage.jsx:45]: Icon is used to provide context that a user can click the button to start downloading their data.

File: src/applications/education-inbox/containers/InboxPage.jsx [hunk 3]
```diff
@@ -1,24 +1,33 @@
 import React, { useEffect, useState } from 'react';
 import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
 import { apiRequest } from 'platform/utilities/api';
-import PropTypes from 'prop-types';
+import moment from 'moment';
```
[the remark this answers is not in the document]
Comment (timwright12) [reply src/applications/education-inbox/containers/InboxPage.jsx:4]: https://date-fns.org/

**Review by timwright12 — APPROVED** [review]

**Review by nathanielbuck — APPROVED** [review]

**Review by cesaresantiago — APPROVED** [review]

**Pull request merged**